RussianFilter/sections/specific.txt

```diff
@@ -6855,7 +6856,7 @@ faqusha.ru##div[class*="advban"]
 gamer.ru##div[class*="alarmalarm-banner-"]
 avito.ru##div[class*="avito-ads-container"] > div
 104.ua##div[class*="b-obozrevatel-"]
-fatalgame.com,kinozz.net,vikka.ua,vinnitsa.info##div[class*="banner"]
+seonews.ru,fatalgame.com,kinozz.net,vikka.ua,vinnitsa.info##div[class*="banner"]
```

Comment:
Not a banner and you block a lot. I'll fix it directly.

Reply:
@gork7777 please see what I meant https://github.com/AdguardTeam/AdguardFilters/commit/e997e4c80a9e4e98a558b4b79f604b305d286188